modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -23,6 +23,21 @@ You can read more about the Infinite Scale beta phase on https://owncloud.com
 - https://owncloud.com/infinite-scale-beta[Infinite Scale Beta Page]
 - https://owncloud.com/news/infinite-scale-beta/[Infinite Scale Beta is coming]
 
+=== Beta 7
+
+Infinite Scale 2.0.0 beta7 includes stability, security and performance improvements. Web Office does now respect the user's language setting and system administrators can recover a lost admin password.
```

Comment:
Web Office does **not** respect the user's language. We can now give the language to the app provider, but we didn't implement it in web, yet. So backend made it possible, but in web ui nothing is different.

Reply:
changed